Modules/CaPTkSurvival/cmdapps/CaPTkSurvivalPredictionCmdApp.cpp

```diff
@@ -108,23 +108,24 @@ int main(int argc, char* argv[])
 
 
     // Handle train/predict mode switching
-    if (us::any_cast<std::string>(parsedArgs["mode"]) == "train")
+    if (us::any_cast<bool>(parsedArgs["type"]) == 0)
     {
         trainNewModel = true;
     }
-    else if (us::any_cast<std::string>(parsedArgs["mode"]) == "predict")
+    else if (us::any_cast<bool>(parsedArgs["type"]) == 1)
     {
         trainNewModel = false;
     }
-    else
+
+    if (us::any_cast<bool>(parsedArgs["type"]))
     {
-        MITK_INFO << parser.helpText();
-        MITK_ERROR << "Provided an invalid mode. See the help text for valid options.";
-        return EXIT_FAILURE;
+        MITK_INFO << "operation type: test";
     }
-
-    MITK_INFO << "mode: " << parsedArgs["mode"].ToString();
-    MITK_INFO << "subjects: " << parsedArgs["subjects"].ToString();
+    else // 0 was passed
+    {
+        MITK_INFO << "operation type: train";
+    }
```

Comment:
This can be simplified

```cpp
    if (us::any_cast<bool>(parsedArgs["type"]))
    {
        trainNewModel = false;
        MITK_INFO << "operation type: test";
    }
    else
    {
        trainNewModel = true;
        MITK_INFO << "operation type: train";
    }
```